api/qsharp/microsoft.quantum.primitive.multix.yml

```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.primitive.multix
 name: MultiX
 type: operation
 namespace: Microsoft.Quantum.Primitive
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  Please use @"microsoft.quantum.canon.applytoeachca".
+summary: MultiX has been deprecated. Please use @"ApplyToEachCA(X, _)" instead.
```

Comment:
@bettinaheim, can you double-check this? Looks like it's not quite the right output.